examples/angular/src/pages/ui/components/authenticator/auth-with-translations/auth-with-translations.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import Amplify, { I18n } from 'aws-amplify';
+import { translations, DefaultTexts } from '@aws-amplify/ui';
```

Comment:
`DefaultTexts` can prob be named better... I didn't use `Translations` because it conflicts with for multi language dictionary `translation` we have. Any ideas are appreciated!